src/main/java/org/apache/sysml/api/mlcontext/MLResults.java

```diff
@@ -330,9 +330,19 @@ public double getDouble(String outputName) {
 		return so.getDoubleValue();
 	}
 
+	public Object get(String outputName) {
```

Comment:
It is meant to be private but exposed on the Python side to avoid using `getDouble` etc. Do you still want me to write documentation?